packages/type-storage/src/wellKnownKeys.ts

```diff
@@ -0,0 +1,56 @@
+import { StorageFunctionModifier, StorageFunctionType } from '@polkadot/api-codec/Metadata';
+import Text from '@polkadot/api-codec/Text';
+import U8a from '@polkadot/api-codec/codec/U8a';
+import Vector from '@polkadot/api-codec/codec/Vector';
+
+import { createFunction } from './utils/createFunction';
+
+interface WKKMetadata {
+  documentation: Vector<Text>;
+  name: Text;
+  type: StorageFunctionType;
+}
+
+// Small helper function to factorize code on this page.
+const createWKKFunction = (prefix: Text | U8a, functionMetadata: WKKMetadata) =>
+  // TODO The expected 2nd argument is a StorageFunctionMetadata, but we actually only need the fields `documentation`
+  // and `type` of it, so in order to not input other fields (byteLength, fromJSON...) we lazily cast as any.
+  createFunction(prefix, { ...functionMetadata, modifier: new StorageFunctionModifier().fromJSON(0) } as any, {
+    isUnhashed: true
+  });
+
+export const code = createWKKFunction(new U8a(':code'), {
+  documentation: new Vector(Text, ['Wasm code of the runtime.']),
+  name: new Text('code'),
+  type: new StorageFunctionType().fromJSON('Vec<u8>')
```

Comment:
How do you create a new StorageFunctionType? A plain type.

Reply:
Well, we should just be able to do `new StorageFunctionType('Vec<u8>')`... However these bloody typed enums are way-over complicated and that doesn't seem to do the trick. :(

Reply:
... in this case, the enum needs to be set and then the value, so it would be standing on one leg, scratching your ears while humming... something like...

```
// init, it by default sets to "Plain"
const type = new StorageFunctionType();

// now set the value on the wrapped type
type.raw.fromJSON('Bytes');
```

(Use `Bytes` here, the types returned are now also aliassed that way, athough I'm guessing the above will do exactly the same actually - if not, it should)

The above is ugly, well, the whole EnumType just feels horrible.